wlm_server/wlm_server/settings.py

```diff
@@ -156,3 +157,6 @@
 MESSAGE_QUEUE = MessageQueue()
 
 CHANNEL_CACHE = None
+
+TASK_LOCK = threading.Lock()
+LOCK_LOCK = threading.Lock()
```

Comment:
In case where the Django uses multiple threads or processes, is it guaranteed that this global assignment is called once, so that these objects are singleton across all the threads and processes? In addition, are other global variables like `MESSAGE_QUEUE` safe?

Reply:
OK, then please add a caution in the `README.md`.